common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -1113,6 +1113,19 @@ void verifyCertificateChain(long[] certificateChainRefs, String authMethod)
                 throws CertificateException;
 
         /**
+         * Verify that we trust the certificate chain is trusted. This is an alternative
+         * to {@link #verifyCertificateChain(long[], String)}, which is given the fully encoded
```

Comment:
That is what I meant, but I've not just removed the old callback as it's no longer being used.